examples/03_connectivity/plot_extract_regions_dictlearning_maps.py

```diff
@@ -32,9 +32,9 @@
 # We use nilearn's datasets downloading utilities
 from nilearn import datasets
 
-adhd_dataset = datasets.fetch_adhd(n_subjects=20)
-func_filenames = adhd_dataset.func
-confounds = adhd_dataset.confounds
+main_dataset = datasets.fetch_main(n_subjects=20)
```

Comment:
On this example, I think that we could increase the number of components from 5 to 8.